.github/workflows/CI.yml

```diff
@@ -93,10 +99,8 @@ jobs:
     name: Deploy preview
     needs: [eslint, typecheck, setup, validate_generate_files, unit_test]
     if: >-
-      (needs.setup.outputs.frontend != 'false' ||
-      needs.setup.outputs.workflows != 'false') &&
-      github.event.pull_request.head.repo.full_name == github.repository &&
-      github.ref_name != 'master'
```

Comment:
This was added so the jobs wouldn't fail because the forks don't have access to our secrets. Would this new job not face the same issues?

Reply:
Oh yes, this was just a merge conflict gone wrong - thanks! :D